kernel/arch/dreamcast/include/dc/scif.h

```diff
@@ -180,12 +182,18 @@ void scif_spi_write_byte(uint8 b);
 /** \brief  Read a byte from the SPI device.
 
     This function reads a byte from the SPI device, one bit at a time. The
-    timing follows that of the scif_spi_rw_byte() function.
+    timing better a bit that of the scif_spi_rw_byte() function.
```

Comment:
Did you mean to say:

timing is a bit better than that of the scif_spi_rw_byte() function.